coco/util.py

```diff
@@ -288,9 +288,10 @@ def sort_list(list_: Dict):
         lists it contains at any depths are sorted. Note that the list and any
         contained lists are not sorted themselves.
     """
-    for i, item in enumerate(list_):
+    list_copy = copy.copy(list_)
```

Comment:
Is this enough? To solve the reference problem I would have thought you need to use a deep copy (i.e. `copy.deepcopy`).